src/interface/contourplot.jl

```diff
@@ -0,0 +1,59 @@
+"""
+    contourplot(x, y, A; kwargs...)
+
+Draws a contour plot on a new canvas.
```

Comment:
I‘m really not a plot power user so it's not immediately clear to me what `x`, `y` and `A` are. Maybe rewrite the description into something like "Draws a contour plot of 2D matrix `A` along axis `x` and `y` on a new canvas."?